src/shared/components/msktable/MSKTable.tsx

```diff
@@ -278,6 +282,60 @@ class MSKTableStore<T> {
     @computed get visibleColumns():Column<T>[] {
         return this.columns.filter(column=>this.isVisible(column));
     }
+
+    @computed get colVisProp(): IColumnVisibilityDef[]
+    {
+        const colVisProp: IColumnVisibilityDef[] = [];
+
+        this.columns.forEach((column:Column<T>) => {
+            colVisProp.push({
+                id: column.name,
+                name: column.name,
+                visible: this.columnVisibility[column.name]
+            });
+        });
+
+        return colVisProp;
+    }
+
+    @computed get textBetweenTheButtons(): string
```

Comment:
`textBetweenPaginationButtons` I think. Or `paginationStatusText` would be even better I think